### ISSUE TITLE
cargoSnapshot uses undefined 'have' command 

### ISSUE THREAD

**Issue by RumataEstor** [NixOS/nixpkgs#7427]
A little after my recent upgrade of rust and accompanying tools i found an error to be emitted during user login process: 'have is not a command' or something similar. I found that command in the bash completion scripts in cargoSnapshot package.

I'd be glad to resolve the issue myself but won't be able to do so for the next 3 weeks. The issue is not critical, so it can wait if nobody is eager to fix it.

**Comment by bjornfor** [comment]
Oh, is that what's causing that annoying little issue?

How come I have this issue on my systems, even though I haven't installed cargoSnapshot?

```
$ ls -l /nix/store/*cargo*
ls: cannot access /nix/store/*cargo*: No such file or directory
```

I've had this issue for a few weeks now, and I'm on the stable branch.

**Comment by peti** [comment]
This an upstream bug. The `have` command is deprecated in the bash-completion package. Instead, `_have` should be used.

**Comment by globin** [comment]
Has been fixed upstream for a while rust-lang/cargo@51b177cac1903876e5eebf1db4656c1700ad4134
